firefox_puppeteer/ui/browser/browserwindow.py

```diff
@@ -0,0 +1,222 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Please rename this file just to `windows.py`. The `browser` part is already part of the folder name.